packages/blocks/src/api/serializer.js

```diff
@@ -261,16 +261,16 @@ export function getCommentDelimitedContent( rawBlockName, attributes, content )
  */
 export function serializeBlock( block ) {
 	const blockName = block.name;
-	const blockType = getBlockType( blockName );
 	const saveContent = getBlockContent( block );
-	const saveAttributes = getCommentAttributes( blockType, block.attributes );
 
 	switch ( blockName ) {
 		case getFreeformContentHandlerName():
 		case getUnregisteredTypeHandlerName():
 			return saveContent;
 
 		default:
```

Comment:
Maybe we should add a block area here `{ }` because to clarify that these consts are local to this default case?